### DISCUSSION TITLE
How can I change the command section that opens the wine programs?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2327] [Q&A]
I mean [that](https://i.imgur.com/uY8EDAn.png),cause I wanna open some programs with other env vars.

**Comment by plata** [comment]
Try enabling the advanced GUI in the settings. This will give you more options when you edit a shortcut in your library.

**Reply by ghost** [reply]
Where can I enable that?
In "User Interface",I can't see something like that there.

**Reply by plata** [reply]
It's in "User Interface": "Advanced mode (requires restart)".

**Reply by ghost** [reply]
Am I blind or something?
Cause I can't find it here:
![image](https://user-images.githubusercontent.com/68649672/105364145-1b6e5080-5bfd-11eb-89fe-4ee0aaff6941.png)

**Reply by plata** [reply]
That's not the latest version.

**Reply by ghost** [reply]
Is t[his package](https://aur.archlinux.org/packages/phoenicis-playonlinux) not the newest version?

**Reply by plata** [reply]
No, see https://github.com/PhoenicisOrg/phoenicis/releases/tag/v5.0-alpha.3.

**Reply by ghost** [reply]
thanks
